### PR TITLE
Improved Czech fiscalization readme.

### PR DESCRIPTION
**Pull request by abdallahbeshi**
image

### CODE REVIEW

File: src/Czechia/README.md [hunk 1]
```diff
@@ -61,45 +98,9 @@ if (response.IsSuccess)
...
+## 🧑 Authors
 Development: [@jirihelmich](https://github.com/jirihelmich)
 Code review: [@siroky](https://github.com/siroky), [@onashackem](https://github.com/onashackem)
 
-# Who uses the library in production?
-- [Mews](https://mews.com) - Property Management Solution for the 21st century.
-- [Hlídač EET](http://hlidaceet.cz) - Watchdog of the EET API endpoint and related services.
-- Keszh, Neszh - [Hart](http://hartphp.com.pl/) Internal ERP System
-- [Ankerdata EasiPOS](http://easipos.ankerdata.com/) - POS system
-- [sySOft](http://www.sysoft.cz/) - POS system
```
Comment (KaliCZ) [inline src/Czechia/README.md:88]: I actually like this part 😄

File: src/Czechia/README.md [hunk 2]
```diff
@@ -61,45 +98,9 @@ if (response.IsSuccess)
...
-There is no need to donate the project, but thanks for considering it! Instead, if you like the project, star it here on GitHub :-)! Thanks!
-
-If you still insist on donating, we accept gummy bears at Mews Systems s.r.o., Náměstí IP Pavlova 5, Vinohrady 120 00 Prague. This project was, of course, powered by a huge pile of gummy bears ;-)
-
-# Related projects
-- [eet-client](https://github.com/todvora/eet-client) by [@todvora](https://github.com/todvora): Java, MIT
-- [SwiftEET](https://github.com/charlieMonroe/SwiftEET) by [@charlieMonroe](https://github.com/charlieMonroe): Swift, GPL-3.0
-
-# Credits
-- [Komodosoft](http://www.komodosoft.net) For publishing [the post](http://www.komodosoft.net/post/2016/03/24/sign-a-soap-message-using-x-509-certificate.aspx) about signing a SOAP message without using WCF.
```
Comment (KaliCZ) [inline src/Czechia/README.md:104]: No credits to Komodosoft? 
No related projects?

File: src/Czechia/README.md (hunk 1 above)
Comment (abdallahbeshi) [reply src/Czechia/README.md:88]: We cannot confirm or deny if that's true :D

File: src/Czechia/README.md (hunk 2 above)
Comment (abdallahbeshi) [reply src/Czechia/README.md:104]: Komodosoft website is down, i already checked 3 days in row.

File: src/Czechia/README.md (hunk 1 above)
Comment (KaliCZ) [reply src/Czechia/README.md:88]: Well somebody did in the past (Jirka). And I believe him until proven incorrect 😂

File: src/Czechia/README.md (hunk 2 above)
Comment (abdallahbeshi) [reply src/Czechia/README.md:104]: related project
eet-client last updated: 17 months ago
SwiftEET: last updated 4 years ago.

Comment (KaliCZ) [reply src/Czechia/README.md:104]: Fair enough

File: src/Czechia/README.md [hunk 3]
```diff
@@ -1,33 +1,70 @@
...
-## Known issues
+## 🎯 Features
+
+-   Functional approach via [FuncSharp](https://github.com/siroky/FuncSharp).
+-   No Czech abbreviations.
+-   Early data validation.
+-   Asynchronous I/O.
+-   All endpoints are covered with tests.
+-   Intuitive immutable DTOs.
+-   Pipelines that run on both Windows and Linux operating systems.
```
Comment (marektresnak) [inline src/Czechia/README.md:46]: I wouldn't list pipelines here, they're not a feature of the library. They are a tool we're using for development of the lib.

**Review by marektresnak — DISMISSED** [review]

**Review by marektresnak — APPROVED** [review]

**Pull request merged**